curriculum.md

```diff
@@ -39,7 +39,41 @@ Here are some basic programming videos that are pretty good, but are done in Pyt
 
 ---
 
-## Week 3: React applications!
+## Week 3: More JavaScript and the DOM
+### Learning objective
+- Complementary JS skills for working with the DOM: 
+   - functions
+   - objects and arrays
+   - events
+   - math functions
+   - this/context
+   - jquery
+   - local storage
+
+### Resources
+- [Front-end JS (Beetroot)](https://beetroot.academy/en/courses/front-end/#module2)
+
+~~Homework~~ (20-40hr practice from Beetroot)
+
+
+---
+
+## Week 4: More JavaScript basics
+### Learning objective
+- Demystifying scopes, closures, regex, es6+, OOP
+
+### Resources
+- [Advanced JavaScript (Beetroot)](https://beetroot.academy/en/courses/advanced-javascript/#module1)
+
+~~Homework~~ (20-32hr practice from Beetroot)
+
+---
+
+## Week 5: No-new-assignment week
+
```

Comment:
Could we add the thoughts that @henriettacooler mentioned here: catch up and maybe expand what we learned so far?

Reply:
I added my suggestion as learning objectives. Please feel free to update them as we find more details for this week!